### DISCUSSION TITLE
Noob question

### DISCUSSION THREAD

**Discussion by chicungunya** [Seji64/SniDust#47] [Q&A] [answered]
Hi,
I have a local proxmox server in Canada. But I need Smart DNS from France.  So I can't use your service right without a VPS in France right? 
I should have a VPS for the Public IP server and the proxmox server as client right?
Thanks!

**Comment by Seji64** [comment]
> I have a local proxmox server in Canada. But I need Smart DNS from France. So I can't use your service right without a VPS in France right?

yep, correct

> I should have a VPS for the Public IP server and the proxmox server as client right?

The client is not relevant. it can be anything where you are able to edit/set the dns server. so yes, your proxmox server can be the client.

**Comment by chicungunya** [comment]
@Seji64 I have my VPS in France. 
If I custom my list and I add tf1.fr (french channel) it works.
But for 6play.fr (french channel) it doesn't, why? 
I added the domains 6play.fr and 6cloud.fr

And for yallo.tv it doesn't too, it is because my VPS is not in Switzerland?

Thank you!

**Comment by Seji64** [comment]
For yallo your vps Must be located in switzerland. Regarding the other channels - sry dont know

**Comment by chicungunya** [comment, marked as answer]
Solved in my custom list with :

tf1.fr
oqee.net
oqee.tv
6play.fr
th2-edge-02.cdn.bedrock.tech

**Reply by Seji64** [reply]
Would you mind to create one (or more) domain lists so i can add it to the repo?

**Reply by chicungunya** [reply]
I will but I'm still blocked with 6play...
I added all the domains of 6play and 6cloud but when I go to this page (private window) : https://www.6play.fr/html5 I still see Canada..
Can you try on your side?

**Reply by Seji64** [reply]
I dont have a VPS with a french IP

**Reply by chicungunya** [reply]
That's fine. If i want to use all the subdomains of 6play.fr and 6cloud.fr on my list I just have to put 6play.fr and 6cloud.fr right? 
Still, I tried with subdomains but it does not work.
I think 6play is hard to by pass. I asked to SmartDNS server https://www.smartdnsproxy.com/ what domains they bypass (because it works with their DNS) but I don't think they will answer to me...
I let you know.

**Reply by Seji64** [reply]
If you think a domain is missing you can enable `SPOOF_ALL_DOMAINS` to verfiy that. But i think they use another mechanism to detect you.
